onnxruntime/core/graph/graph.cc

```diff
@@ -2260,6 +2265,10 @@ Status Graph::VerifyNodeAndOpMatch(const ResolveOptions& options) {
   ctx.set_ir_version(gsl::narrow_cast<int>(IrVersion()));
   ctx.set_opset_imports(DomainToVersionMap());
   ctx.set_schema_registry(schema_registry_.get());
+  // Set the parent directory of model path to load external tensors if exist
+  const auto& dir_path = ModelPath().ParentPath().ToPathString();
+  std::string dir_path_str(dir_path.begin(), dir_path.end());
+  ctx.set_model_dir(dir_path_str + "\\");
 
```

Comment:
Does this work on linux or should we use "/"?

Reply:
oh my bad! good catch I added this to run some local tests... Jacky has a PR to fix this at this time it was not checked in so I added this temp code...